agent/agentssh/agentssh.go

```diff
@@ -1082,15 +1103,36 @@ func (s *Server) Close() error {
 	s.closing = nil
 	s.mu.Unlock()
 
+	s.logger.Debug(ctx, "closing server done")
+
 	return err
 }
 
-// Shutdown gracefully closes all active SSH connections and stops
+// Shutdown ~~gracefully~~ closes all active SSH connections and stops
```

Comment:
```suggestion
// Shutdown forcefully closes all active SSH connections and stops
```

Reply:
I'd like to avoid manifesting the current implementation as the final intent. I can change the wording but I don't want to make it seem like Shutdown is supposed to be forceful.

Reply:
Fair!